requirements.txt

```diff
@@ -1,7 +1,7 @@
 xlrd==1.2.0
 #python-magic==0.4.15 #in ckancore
 messytables==0.15.2
-progressbar2==4.0.0
+progressbar==2.5
```

Comment:
could this have done it?

Reply:
I don't think so. The errors that I was seeing looked quite unrelated; CKAN was failing to find an attribute on one of its own modules, despite that attribute being present.